dcat3-implementation-report/index.html

```diff
@@ -12,14 +12,170 @@
+<p>Evidence of implementation was gathered from existing datasets, which already implement the proposed changes, as well as from existing application profiles of [[VOCAB-DCAT]].</p>
+      </section>
+        </section>  
+<section id="proposed-revisions">
+      <h2>Summary of proposed revisions</h2>
+      <p>The following table shows the set of changes proposed in [[VOCAB-DCAT-3]].</p>
+
+<aside class="note">
+<p> <code>dcat:Resource</code> is an abstract class that is not supposed to be used directly, as explained in <a data-cite="VOCAB-DCAT-2#dcat-scope">&sect;&nbsp;<span class="secno">5.1 </span>DCAT scope</a> of [[VOCAB-DCAT-2]]. The properties are considered when inherited by a sub-class of <code>dcat:Resource</code>.</p>
+<!--
+<p>Quoting:</p>
+<blockquote>
+<p><a data-cite="VOCAB-DCAT-2#Class:Resource"><code>dcat:Resource</code></a> represents a dataset, a data service or any other resource that may be described by a metadata record in a catalog. This class is not intended to be used directly, but is the parent class of <a data-cite="VOCAB-DCAT-2#Class:Dataset"><code>dcat:Dataset</code></a>, <a data-cite="VOCAB-DCAT-2#Class:Data_Service"><code>dcat:DataService</code></a> and <a data-cite="VOCAB-DCAT-2#Class:Catalog"><code>dcat:Catalog</code></a>. Member items in a catalog should be members of one of the sub-classes, or of a sub-class of these, or of a sub-class of <a data-cite="VOCAB-DCAT-2#Class:Resource"><code>dcat:Resource</code></a> defined in a DCAT profile or other DCAT application. <a data-cite="VOCAB-DCAT-2#Class:Resource"><code>dcat:Resource</code></a> is effectively an extension point for defining a catalog of any kind of resource. <a data-cite="VOCAB-DCAT-2#Class:Dataset"><code>dcat:Dataset</code></a> and <a data-cite="VOCAB-DCAT-2#Class:Data_Service"><code>dcat:DataService</code></a> can be used for datasets and services which are not documented in any catalog.</p>
+</blockquote>
+-->
+</aside>
+
+<div data-include="table-revisions.html"></div>
+            
+</section>
+<section id="implementations">
+        <h3>Implementations</h3>
+<section id="vocabularies">
+        <h3>DCAT-based vocabularies and datasets</h3>
+        <p>The following table shows DCAT-based vocabularies providing implementation evidence of the revisions proposed in [[VOCAB-DCAT-3]].</p>
+
+        <table id="table-vocabularies" class="simple">
+          <thead>
+            <tr>
+              <th>ID</th>
+              <th>Application Profile/Datasets in Catalog</th>
+              <th>Creator</th>
+              <th>Country</th>
+              <th>Year</th>
+            </tr>
+          </thead>
+          <tbody>
+               <tr>
+              <td id="V02">V01</td>
+              <td>[[DCAT-AP-3]]</td>
+              <td>SEMIC action, Interoperable Europe, European Commission</td>
+              <td>Europe</td>
+              <td>2024</td>
+            </tr>
+              <tr>
```

Comment:
I think DCAT-EP and GeoDCAT-AP has been swapped. I.e.:
- V02 should be GeoDCAT-AP
- V03 should be DCAT-EP

Reply:
Thanks, Andrea 
https://github.com/w3c/dxwg/pull/1594/commits/97814c20884d189229588d36d37c9e312ae53bf4 should have fixed this.